.github/workflows/performance-regression.yml

```diff
@@ -0,0 +1,146 @@
+name: Performance Regression Detection
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
We might want to trigger the workflow only when the source code of `ion-c` has been changed. 
```
on:
pull_reuqest: 
  paths:
        - 'ionc/*'
```

Reply:
Good call, adding.